src/application/service/config/GetServiceConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { ServiceConfigDTO } from '../../../presentation/dtos/config/ServiceConfigDTO';
+
+export class GetServiceConfig {
+  constructor(private config: any) {}
+
+  public async execute(): Promise<ServiceConfigDTO> {
+    return {
+      allowedAuthMethods: this.config.get('authentication.allowedMethods'),
```

Comment:
Could we only expose one `authenticationMethod` for now, as we only have a use case with 1 method per deployment, and it means we wouldn't have to figure out logic on the clients for which method is preferable (to choose the login page and test identifier, for example).

Reply:
Wouldn't want to unnecessarily complicate the client logic on multiple pages for a use case we're not sure is ever going to be there.

Reply:
i think it'd be better if you do a authMethods[0] in the client rather than having to version the API later on

Reply:
Although I'm not entirely convinced, we can do that indeed. We then have to make sure the order coming from the server is different depending on the deployment, though.

Reply:
So maybe it's easier to just expose `authenticationMethod` for now?